modules/environment/default.nix

```diff
@@ -206,5 +219,8 @@ in
       ${concatStringsSep "\n" aliasCommands}
     '';
 
+    warnings = lib.optional (
+      cfg.postBuild != ""
+    ) "The option `environment.postBuild` has been renamed to `environment.extraSetup`";
```

Comment:
```suggestion
```

This should use `mkRenamedOption` instead

Reply:
Didn't find that in docs, will do